src/main/java/rs/teslaris/core/util/notificationhandling/NotificationFactory.java

```diff
@@ -0,0 +1,49 @@
+package rs.teslaris.core.util.notificationhandling;
+
+import java.util.Map;
+import rs.teslaris.core.model.commontypes.Notification;
+import rs.teslaris.core.model.commontypes.NotificationType;
+import rs.teslaris.core.model.user.User;
+
+public class NotificationFactory {
+
+    public static Notification contructNewOtherNameDetectedNotification(
+        Map<String, String> notificationValues, User user) {
+        switch (user.getPreferredLanguage().getLanguageCode()) {
+            case "SR":
+                return new Notification(
+                    "Dodati ste na publikaciju sa novim oblikom imena (" +
```

Comment:
This is a bad practice to have here i18n messages which are going to UI

Reply:
I will now try to migrate this to use Spring's MessageSource.

Reply:
Done :smile: :rocket: